src/util/events.js

```diff
@@ -81,4 +81,18 @@ export function computeFilteredEvents(events, filters, zipcodes) {
 
     return true;
   });
+
+  // When a zipcode is selected, sort events by proximity to that zipcode.
+  if (filters.zipcode) {
+
+    filteredEvents.sort((a, b) => {
```

Comment:
...but FWIW this is not the original, canonical data source being mutated, it is a new list generated within this function.